csrc/cpu/cpu_attn_impl.hpp

```diff
@@ -1246,14 +1246,8 @@ class AttentionMainLoop {
         // rescale sum and partial outputs
         if (need_rescale) {
           // compute rescale factor
-#ifdef DEFINE_FAST_EXP
-          vec_op::FP32Vec16 rescale_factor_vec(rescale_factor);
-          rescale_factor_vec = fast_exp(rescale_factor_vec);
-          rescale_factor = rescale_factor_vec.get_last_elem();
-#else
```

Comment:
Shouldn't remove the fast exp path.

Reply:
This is calculating the exp of a scalar. 
Broadcasting it and using fastexp over 16 vector lanes is certainly and overkill, and a disadvantage for most CPUs without 512 vector length - e.g. Arm Neon has 128 bit vector length.

We can add an ifdef to go to this fastexp for x86 but IMHO it’s a bad idea. Whatever few cycles this will save you won’t be worth making the code uglier.